.github/workflows/packages.yml

```diff
@@ -1,11 +1,24 @@
 name: Build Mergin Plugin Packages
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      MERGIN_CLIENT_VER:
+        description: 'Mergin client version either a tag, release, or branch'
+        required: true
+        default: 'master'
+        type: string
+      GEODIFF_VER:
+        description: 'Geodiff version released on PyPI repository'
+        required: true
+        default: '2.0.4'
+        type: string
```

Comment:
I'd probably mark the geodiff as not required, not sure how often we'd need to do that :)

Reply:
You right not used much, removed required in latest